docs/source/model_design/risk_factors.rst

```diff
@@ -0,0 +1,94 @@
+Relative Risk (RR)
+++++++++++++++++++
+
+Population Attributable Fraction (PAF)
+++++++++++++++++++++++++++++++++++++++
+
+Description
++++++++++++
+
+PAF in GBD 2017 (operationally)
++++++++++++++++++++++++++++++++
+
+Defintion of PAF
+++++++++++++++++
+
+The population attributable fraction (PAF) represents the proportion of risk that would be reduced in a given year if the exposure to a risk factor in the past were reduced to an ideal exposure scenario. This is defined for a continuous risk factor as:
```

Comment:
Double check something didn't get lost in copy + paste. This looks like it's missing an equation.

Reply:
Reframe to not only define for a continuous risk factor. This is misleading and causing confusion.

Reply:
To-do: For RR, how would this be framed for non-continuous RF?

Reply:
Add information in 'ideal exposure scenario' about linking to TMRL.

Reply:
Replace 'proportion of risk' to 'proportion of outcome' or 'disease'.